ci/test-conformance-gke.sh

```diff
@@ -246,7 +260,6 @@ function cleanup_cluster() {
        echo "=== Failed to delete GKE cluster ${CLUSTER}! ==="
        exit 1
     fi
-    rm -f $HOME/.kube/kubeconfig
```

Comment:
why this change and why for GKE only?

Reply:
This was done because I wanted to retain GKE cluster config for debugging. Added back.